save-backend/src/main/kotlin/org/cqfn/save/backend/repository/TimestampBasedFileSystemRepository.kt

```diff
@@ -42,6 +48,11 @@ class TimestampBasedFileSystemRepository(configProperties: ConfigProperties) {
             createDirectories()
         }
     }
+    private val rootDirImage = (Paths.get(configProperties.fileStorage.location) / "imageAvatar").apply {
```

Comment:
```suggestion
    private val rootDirImage = (Paths.get(configProperties.fileStorage.location) / "images" / "avatars").apply {
```
Imagine, how many top-level directories will there be with naming like this

Reply:
Fixed